api/app/src/main/kotlin/packit/service/UserService.kt

```diff
@@ -47,6 +51,41 @@ class BaseUserService(
         return newUser
     }
 
+    override fun createBasicUser(createBasicUser: CreateBasicUser)
+    {
+        val existingUser = userRepository.findByUsername(createBasicUser.email)
+        if (existingUser != null)
+        {
+            throw PackitException("userAlreadyExists", HttpStatus.BAD_REQUEST)
+        }
+
+        val foundUserGroups = getFoundUserGroups(createBasicUser)
+        val newUser = User(
+            username = createBasicUser.email,
+            displayName = createBasicUser.displayName,
+            disabled = false,
+            email = createBasicUser.email,
+            userSource = "basic",
+            lastLoggedIn = Instant.now().toString(),
```

Comment:
I think this should be null - we're just creating the user here, not actually logging them in. 
Some users will never actually log in themselves, and it's useful to be able to see that. 
Though maybe we need to think more about what the process of creating a user is and where the initial password comes from.